internal/service/alarms/internal/db/repo/alarms_repository.go

```diff
@@ -21,16 +23,21 @@ type AlarmsRepository struct {
 // GetAlarmEventRecordWithUuid grabs a row of alarm_event_record using uuid
 func (ar *AlarmsRepository) GetAlarmEventRecordWithUuid(ctx context.Context, uuid uuid.UUID) ([]models.AlarmEventRecord, error) {
 	// Build sql query
-	query := `
-SELECT alarm_event_record_id, alarm_definition_id, probable_cause_id, alarm_raised_time, 
-       alarm_changed_time, alarm_cleared_time, alarm_acknowledged_time, 
-       alarm_acknowledged, perceived_severity, extensions, resource_type_id
-FROM alarm_event_record 
-WHERE alarm_event_record_id = $1
-`
+	query, args, err := psql.Select(
+		sm.Columns(
+			AlarmEventRecordID, AlarmDefinitionID, ProbableCauseID,
```

Comment:
Would be good if we could somehow get this list automatically from the model object in the `alarms.internal.db.models` package.

Reply:
+1 for that! Hope there's way from Bob to easily convert that? Otherwise we might look into using reflection and just extract the "db" tag values?

Reply:
Yes, reflection is the only option. I'll give it a try tomorrow. If it is too messy I'll stick with this

Reply:
Looks like we leverage the codegen from the lib https://bob.stephenafamo.com/docs/code-generation/usage (looks really nice tbh)

Reply:
Done, please have another look @pixelsoccupied @alegacy